python/src/MarginalUniformOrderStatistics_doc.i.in

```diff
@@ -0,0 +1,109 @@
+%feature("docstring") OT::MarginalUniformOrderStatistics
+"Marginal uniformOrderStatistics.
+
```

Comment:
the experimental module warning is missing

Reply:
Fixed